src/main/resources/schema.sql

```diff
@@ -0,0 +1,47 @@
+
+CREATE DATABASE IF NOT EXISTS airlines;
+CREATE USER 'user'@'localhost' IDENTIFIED BY 'user';
+GRANT ALL PRIVILEGES ON airlines.* TO 'user'@'localhost';CREATE DATABASE IF NOT EXISTS airlines;
+CREATE USER 'user'@'localhost' IDENTIFIED BY 'user';
```

Comment:
You have duplication:
```
CREATE DATABASE IF NOT EXISTS airlines;
CREATE USER 'user'@'localhost' IDENTIFIED BY 'user';
GRANT ALL PRIVILEGES ON airlines.* TO 'user'@'localhost';
```
repeats twice.
It should be once

Reply:
fixed in